packages/boot/test/bootstrapTests/ec-membership-update.test.ts

```diff
@@ -401,7 +411,262 @@ test.serial(
+  const outgoingMember = governanceDriver.ecMembers[3];
+
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(storage);
+  const { VaultFactory } = agoricNamesRemotes.instance;
+  const { ATOM: collateralBrand, IST: debtBrand } = agoricNamesRemotes.brand;
+
+  t.log('Proposing question using old charter invitation');
+  await governanceDriver.proposeParams(
+    VaultFactory,
+    { DebtLimit: { brand: debtBrand, value: 300_000_000n } },
+    { paramPath: { key: { collateralBrand } } },
+    outgoingMember,
+    getQuestionId(3),
+    offerIds.propose.outgoing,
+  );
+
+  t.like(outgoingMember.getLatestUpdateRecord(), {
+    status: { id: getQuestionId(3), numWantsSatisfied: 1 },
+  });
+
+  t.log('Voting on question using first 2 wallets');
+  await governanceDriver.enactLatestProposal(
+    newCommittee.slice(0, 2),
+    getVoteId(3),
+    offerIds.vote.incoming,
+  );
+  for (const w of newCommittee.slice(0, 2)) {
+    t.like(w.getLatestUpdateRecord(), {
+      status: { id: getVoteId(3), numWantsSatisfied: 1 },
+    });
+  }
+
+  t.log('Waiting for period to end');
+  await advanceTimeBy(1, 'minutes');
+
+  t.log('Verifying outcome');
+  const lastOutcome = await governanceDriver.getLatestOutcome();
+  const managerParams = getVstorageData(managerGovernanceKey);
+  t.deepEqual(managerParams.current.DebtLimit.value.value, 300_000_000n);
+  t.assert(lastOutcome.outcome === 'win');
+});
+
+test.serial('successful proposal and vote for auctioneer', async t => {
+  const { storage, advanceTimeBy, getVstorageData, governanceDriver } =
+    t.context;
+  const newCommittee = governanceDriver.ecMembers.slice(0, 3);
+
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(storage);
+  const { auctioneer } = agoricNamesRemotes.instance;
+
+  t.log('Proposing question using new charter invitation');
+  await governanceDriver.proposeParams(
+    auctioneer,
+    { LowestRate: 100_000_000n },
+    { paramPath: { key: 'governedParams' } },
+    newCommittee[0],
+    getQuestionId(4),
+    offerIds.propose.incoming,
+  );
+
+  t.like(newCommittee[0].getLatestUpdateRecord(), {
+    status: { id: getQuestionId(4), numWantsSatisfied: 1 },
+  });
+
+  t.log('Voting on question using first 2 wallets');
+  await governanceDriver.enactLatestProposal(
+    newCommittee.slice(0, 2),
+    getVoteId(4),
+    offerIds.vote.incoming,
+  );
+  for (const w of newCommittee.slice(0, 2)) {
+    t.like(w.getLatestUpdateRecord(), {
+      status: { id: getVoteId(4), numWantsSatisfied: 1 },
+    });
+  }
+
+  t.log('Waiting for period to end');
+  await advanceTimeBy(1, 'minutes');
+
+  t.log('Verifying outcome');
+  const lastOutcome = await governanceDriver.getLatestOutcome();
+  const auctioneerParams = getVstorageData(auctioneerParamsKey);
+  t.deepEqual(auctioneerParams.current.LowestRate.value, 100_000_000n);
+  t.assert(lastOutcome.outcome === 'win');
+});
+
+test.serial('successful proposal and vote for provisionPool', async t => {
```

Comment:
```suggestion
test.serial('EC can govern provisionPool parameter', async t => {
```

This starts to get repetitive. You might try a `test.macro` parameterized on contract instance and governed parameter.

Reply:
The issue calls for checking all instanced of 6 contract installations. I see only 5. Reserve seems to be missing.

But maybe it's there and I don't see it. It would be nice to see all 6 on one screen using invocations of a macro. For example:

```js
const ecCanGovern = test.macro({
  title:(_t, label) => `EC can govern ${label}`,
  exec(t, label, findInstances, parameters) => { ... },
});

test('auctioneer', ecCanGovern, instance => [instance.auctioneer], { LowestRate: 100_000_000n });
test('reserve', ecCanGovern, instance => [instance.reserve], { ... });
test('PSM', ecCanGovern, instance => instance.map(...).filter(...), { ... });
```

I hope that's enough of a suggestion. What do you think?

Reply:
added the test for reserve. missed that accidentally. thanks for the catch

Reply:
regarding `test.macro`, i tried it out. personally it didn't improve readability a lot since there are a lot of minor variations between the tests due to which I have to pass a ton of parameters to the macro. reading the tests at first glance, it isnt immediately obvious what those parameters do

Reply:
thanks for trying it out.